CollapsibleExample/ExampleSubclassFilterViewController.m

```diff
@@ -11,6 +11,7 @@
 @interface ExampleSubclassFilterViewController ()
 
 - (NSArray*)returnLabelArray;
```

Comment:
So I take it when I subclass for missionhub this would be the best practice after getting data back from the API?